### PR TITLE
feat(errors): report errors with x-error header

### PR DESCRIPTION
**Pull request by trieloff**

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -47,6 +47,7 @@ function error(message, code = 500) {
       'Content-Type': 'text/html',
       'X-Static': 'Raw/Static',
       'Cache-Control': 'max-age=300',
+      'X-Error': message,
```
Comment (tripodsan) [inline src/utils.js:50]: actually..... I'm not a big fan of reporting errors back to the client in a visible manner.
we need to take extra care not to reveal any _secrets_. we have so many logs...it should be possible to automatically grab those, like with @kptdobe 's  trace-and-debug

File: src/handlers/font-css.js [hunk 2]
```diff
@@ -96,6 +96,7 @@ async function deliverFontCSS({ params: { kitid } }) {
     log.error(`Error while retrieving font: ${e.message}`);
     return new Response('', {
       status: 502,
+      'x-error': e.message,
```
Comment (tripodsan) [inline src/handlers/font-css.js:99]: don't return the plain message.

File: src/utils.js (hunk 1 above)
Comment (tripodsan) [reply src/utils.js:50]: ok, so why not using the `sanitizer.escape(message)` ?

Comment (trieloff) [reply src/utils.js:50]: Because there is no XSS risk in HTTP headers.

**Review by tripodsan — APPROVED** [review]
just make sure, the error message doesn't contain new lines, otherwise setting the error throws an exception.

**Pull request merged**